src/main/java/frc/robot/Subsystems/ClimberPlacer.java

```diff
@@ -1,50 +1,68 @@
 package frc.robot.Subsystems;
 
-import com.spikes2212.command.genericsubsystem.MotoredGenericSubsystem;
 import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.spikes2212.command.genericsubsystem.MotoredGenericSubsystem;
 import edu.wpi.first.wpilibj.DigitalInput;
 import frc.robot.RobotMap;
 
+import java.util.function.Supplier;
+
 /**
  * Controls the position of the climber.
  */
 public class ClimberPlacer extends MotoredGenericSubsystem {
 
     private static final double MIN_SPEED = -0.6;
     private static final double MAX_SPEED = 0.6;
+    private static ClimberPlacer rightInstance, leftInstance;
 
-    private static ClimberPlacer instance;
+    private final Supplier<Double> dropSpeed = rootNamespace.addConstantDouble("drop speed", 0);
 
     private final DigitalInput frontLimit;
     private final DigitalInput backLimit;
-    private static WPI_TalonSRX leftPlacer;
-    private static WPI_TalonSRX rightPlacer;
-
-    public static ClimberPlacer getInstance() {
-        if (instance == null) {
-            leftPlacer = new WPI_TalonSRX(RobotMap.CAN.PLACER_TALON_LEFT);
-            rightPlacer = new WPI_TalonSRX(RobotMap.CAN.PLACER_TALON_RIGHT);
-            instance = new ClimberPlacer(leftPlacer, rightPlacer);
+    private final WPI_TalonSRX placer;
+
+    public static ClimberPlacer getRightInstance() {
```

Comment:
switch the order of getLeftInstance() and getRightInstance() (Convention A.III.b.1)